packages/material-ui/src/AvatarGroup/AvatarGroup.js

```diff
@@ -14,15 +14,6 @@ const SPACINGS = {
   medium: null,
 };
 
-const overridesResolver = (props, styles) => {
-  return deepmerge(
-    {
-      [`& .${avatarGroupClasses.avatar}`]: styles.avatar,
```

Comment:
This is a good example where this approach may not work everywhere. In this case we need to still keep this, otherwise it won't work.